stable/redis-ha/values.yaml

```diff
@@ -66,9 +60,13 @@ persistentVolume:
   ##
   subPath: ""
 ## Redis image version
-redis_image: quay.io/smile/redis:4.0.2
+redis_image: quay.io/ianmaddox/redis-sentinel:redis-4.0.6r0
 ## replicas number for each component
 replicas:
-  master: 1
-  slave: 1
-  sentinel: 3
+  servers: 3
+  sentinels: 3
+appliance:
```

Comment:
What is meant by appliance here? Can you add some info in the readme about how/when to configure like this?

Reply:
Good catch. I put all the documentation in the Dockerfile for the image. Added helpful notes to README.